dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -1927,6 +1927,97 @@ def test_unique_row_ratio_empty_profiler(self):
         profiler = StructuredProfiler(pd.DataFrame([]))
         self.assertEqual(0, profiler._get_unique_row_ratio())
 
+    def test_null_replication_metrics_calculation(self):
+        data = pd.DataFrame(
+            {
+                "a": [3, 2, np.nan, 7, np.nan],
```

Comment:
should we validate that the clean, etc. works for nulls that aren't just caught by `np.nan`. e.g. blank spaces, or None?

Reply:
As long as the DataProfiler successfully detects null values the calculation shouldn't be affected. Could definitely add that to the tests though.